lib/computer.ex

```diff
@@ -3,53 +3,97 @@ defmodule Computer do
   defp other_mark(:o), do: :x
 
   def move(board, mark) do
+    if board == Board.empty_board do
+      random_move()
+    else
+      board |> most_valuable_move(mark)
+    end
+  end
+
+  def random_move() do
+    Enum.take_random([0, 2, 4, 6, 8], 1) |> Enum.at(0)
+  end
+
+  defp most_valuable_move(board, mark) do
     board
     |> Board.available_placeholders
-    |> Enum.map(fn(x) ->
-      {x, minimax(Board.set_mark(board, mark, x), mark, false, 9)} end)
-    |> Enum.max_by(fn(x) -> elem(x, 1) end)
-    |> elem(0)
+    |> try_all_moves(board, mark)
+    |> select_best_move
   end
 
-  defp minimax(board, mark) do
-    minimax(board, mark, false, 9)
+  def try_all_moves(positions_available, board, mark) do
+    Enum.map(
+      positions_available,
+      fn(position) ->
```

Comment:
I don't think this reads particularly well...
position
|> Enum.map(...)

And then see if you can shrink that function? `false` and `9` seem to be constants!